content/doc/book/security/access-control/permissions.adoc

```diff
@@ -107,6 +105,101 @@ Anything they cannot accomplish through the existing UI they can do through the
 An instance on which an untrusted user gained _Overall/Administer_ permission should be considered fully compromised and should be replaced.
 All secrets (credentials, etc.) stored on that instance should be rotated or revoked and all artifacts created from that point on should be verified.
 
+
+=== Agent permissions
+
+==== Agent/Build
+
+This permission allows users to run jobs as them on agents.
+
+==== Agent/Configure
+
+This permission allows users to configure agents.
+
+==== Agent/Connect
+
+This permission allows users to connect agents or mark agents as online.
+
+This permission is implied by Agent/Disconnect.
+
+==== Agent/Create
+
+This permission allows users to create agents.
+
+==== Agent/Delete
+
+This permission allows users to delete existing agents.
+
+==== Agent/Disconnect
+
+This permission allows users to disconnect agents or mark agents as temporarily offline.
+
+=== Job permissions
+
+==== Job/Build
+
+This permission grants the ability to start a new build.
+
+==== Job/Cancel
+
+This permission grants the ability to cancel a scheduled, or abort a running, build.
+
+==== Job/Configure
+
+Change the configuration of a job.
+
+==== Job/Create
+
+Create a new job.
+
+==== Job/Delete
+
+Delete a job.
+
+==== Job/Discover
+
+This permission grants discover access to jobs. Lower than read permissions, it allows you to redirect anonymous users to the login page when they try to access a job url.
+Without it they would get a 404 error and wouldn't be able to discover project names.
+
+This permission is implied by Job/Read.
+
+==== Job/Move
+
+Required to move a job from one folder (or Jenkins root) to another.
+
+==== Job/Read
+
+See a job. (You may deny this permission but allow Discover to force an anonymous user to log in to see the job.)
+
+==== Job/Workspace
+
+This permission grants the ability to retrieve the contents of a workspace Jenkins checked out for performing builds.
+If you don’t want a user to access files in the workspace (e.g. source code checked out from SCM or intermediate build results) through the workspace browser, you can revoke this permission.
+
+=== Run permissions
+
+==== Run/Delete
+
+This permission allows users to manually delete specific builds from the build history.
+
+==== Run/Update
+
+This permission allows users to update description and other properties of a build, for example to leave notes about the cause of a build failure.
+
+=== View permissions
+
+==== View/Configure
+This permission allows users to change the configuration of views.
+
+==== View/Create
+This permission allows users to create new views.
+
+==== View/Delete
+This permission allows users to delete existing views.
+
+==== View/Read
+This permission allows users to see views (implied by generic read access).
```

Comment:
> implied by generic read

That's a permission that exists but hasn't been shown on the UI pretty much ever, so it's not particularly helpful.